### PR TITLE
feat(features/cards): loading skeleton cards

### LINKED ISSUE #50: Show some loading indicator when cards are loading.
Due to slow connection users might think there are no cards, meanwhile they just haven't been loaded yet.
HowtoCards 2019-05-20 16-28-16

### PR DESCRIPTION
**Pull request by meowsun**
Closes #50 
Show some loading indicator when cards are loading.

### CODE REVIEW

**Comment by meowsun** [comment]
Closes #50 (Show some loading indicator when cards are loading. )

**Comment by sergeysova** [comment]
[Ревью skeleton cards loading](https://trello.com/c/vFyVLYHC/3-%D1%80%D0%B5%D0%B2%D1%8C%D1%8E-skeleton-cards-loading)

File: src/features/cards/organisms/cards-list.js [hunk 1]
```diff
@@ -18,20 +20,31 @@ export const CardsList = ({ ids, renderCard }) => {
...
-      renderExists={(list) => (
-        <CardsItemsBlock>
-          {list
-            .filter(Boolean)
-            .map((card) =>
-              renderCard({ card, onUsefulClick: () => onUsefulClick(card.id) }),
-            )}
-        </CardsItemsBlock>
+    <>
+      {isLoading && (
```
Comment (sergeysova) [inline src/features/cards/organisms/cards-list.js]: Use ternary operator here.

```js
{isLoading
	? ( SkeletonHere )
	: ( ConditionalList here )
}
```

**Comment by sergeysova** [comment]
@meowsun 	Thank you! 💜

**Pull request merged**